custom-connectors/PUGGamifiedEngagement/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "INTEGRATION API KEY",
+          "description": "INTEGRATION API KEY para esta API.",
```

Comment:
@smart-jp Non-english characters are not allowed in any connector artifacts, Can you please remove them or replace them with English language characters